backend/Car-Detailing-Backend/obj/Car-Detailing-Backend.csproj.nuget.dgspec.json

```diff
@@ -0,0 +1,75 @@
+{
+  "format": 1,
+  "restore": {
+    "C:\\Users\\marci\\OneDrive\\Pulpit\\Cyberbezpieczenstwo\\TPF-03-Proj\\Car-Detailing\\backend\\Car-Detailing-Backend\\Car-Detailing-Backend.csproj": {}
+  },
+  "projects": {
+    "C:\\Users\\marci\\OneDrive\\Pulpit\\Cyberbezpieczenstwo\\TPF-03-Proj\\Car-Detailing\\backend\\Car-Detailing-Backend\\Car-Detailing-Backend.csproj": {
+      "version": "1.0.0",
+      "restore": {
+        "projectUniqueName": "C:\\Users\\marci\\OneDrive\\Pulpit\\Cyberbezpieczenstwo\\TPF-03-Proj\\Car-Detailing\\backend\\Car-Detailing-Backend\\Car-Detailing-Backend.csproj",
+        "projectName": "Car-Detailing-Backend",
+        "projectPath": "C:\\Users\\marci\\OneDrive\\Pulpit\\Cyberbezpieczenstwo\\TPF-03-Proj\\Car-Detailing\\backend\\Car-Detailing-Backend\\Car-Detailing-Backend.csproj",
```

Comment:
I don't think files like that should be found here. Please add `.gitignore` file to remove them from git history :sweat_smile:

Reply:
All junk files have been removed if any still appear I will respond to this on an ongoing basis